definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4378,7 +4385,7 @@
     "LiveFrom": "01/03/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorApplyForDN",
-    "UserRole": "caseworker-divorce-solicitor",
+    "UserRole": "[PETSOLICITOR]",
```

Comment:
Is this an actual user role?

Reply:
This shouldn't be going to prod yet, until the respondent solicitor release is ready to go.